### DISCUSSION TITLE
LobeChat desktop & mobile application?

### DISCUSSION THREAD

**Discussion by Asuka109** [lobehub/lobehub#451] [General]
Excellent works!

How about introducing Lobe Chat as an application on both desktop and mobile platforms? This will enable users to interact with agents in various new ways.
Users can provide context for the conversation by sharing text/files within the mobile app, or they can use shortcuts to launch the desktop application and translate selected text.

Hope to contribute!

**Comment by canisminor1990** [comment]
@Asuka109 

The application is planned for our 1.0 release, but in the meantime, we are focusing on enhancing the agent promotion capabilities and expanding the plugin ecosystem for FC. As for the application, there are still many points to discuss, such as whether to use Electron for packaging or switch to Tauri for smaller bundle sizes. If you have experience in this area, we would love to have you join the discussion and contribute! ❤️

---

Application 在我们 1.0 后期的规划中，但前期我们主要会丰富 agent promot 的构建能力和 FC 的插件生态，同时关于 application 目前还遗留着非常多的待讨论点，比如使用 electron 封装还是用 tauri 换取更小的打包体积。如果你有这方面的经验，非常欢迎参与讨论和贡献~ ❤️

**Reply by Asuka109** [reply]
@canisminor1990  Cool! I have never used Electron or Tauri to develop production-ready applications before, but I am very interested in it. I learn that Electron has some solutions (Capacitor and Ionic) for building mobile applications, but I'm not sure if these solutions are mature and popular enough to be used.
On the other hand, Tauri is soon going to support mobile development in its 2.0 version, and besides the advantage of smaller bundle size, I am particularly excited about Rust's portability in cross-platform applications. After the beta release of Tauri 2.0, I will experiment with integrating lobechat into Tauri mobile apps.

As for now, I would like to contribute to Agents and FC. Is there anything that I need to know?

**Reply by canisminor1990** [reply]
@Asuka109 

- Readme: https://github.com/lobehub/lobe-chat
- Wiki: https://github.com/lobehub/lobe-chat/wiki
- Plugin Development: https://github.com/lobehub/lobe-chat/wiki/Plugin-Development
- Plugin Index: https://github.com/lobehub/lobe-chat-plugins
- Agent Index: https://github.com/lobehub/lobe-chat-agents
